sklearn/ensemble/_hist_gradient_boosting/tests/test_gradient_boosting.py

```diff
@@ -820,7 +820,7 @@ def test_unknown_categories_nan(insert_missing, Est,
         categorical_features = [1]
 
     if insert_missing:
-        mask = rng.binomial(1, 0.01, size=X.shape).astype(np.bool)
+        mask = rng.binomial(1, 0.01, size=X.shape).astype(np.bool_)
```

Comment:
[We agree to use](https://github.com/scikit-learn/scikit-learn/pull/17875) the built-in `bool` type:

```suggestion
        mask = rng.binomial(1, 0.01, size=X.shape).astype(bool)
```